### ISSUE TITLE
Change UX for Delete Properties of Schema

### ISSUE THREAD

**Issue by sherl0cks** [Apicurio/apicurio-studio#344]
Currently the delete all properties button is next to the add property button. It's easy to click the wrong one, and the there is no confirmation prompt. Easy way too lose a lot of time. See bottom right of screenshot

![screenshot from 2018-07-17 23-49-00](https://user-images.githubusercontent.com/1335137/42862269-03a4e064-8a1c-11e8-8f01-a695f8e25d79.png)

**Comment by EricWittmann** [comment]
This is a fair point, and something that Monica has called out as well.  I'm open to changing it, obviously.  But I have a question for you:  if **Undo/Redo** were available, would that fix the issue?

So if you clicked on the wrong thing you could just Ctrl-Z to recover.

**Comment by sherl0cks** [comment]
Not sure it would eliminate the issue, but it would help a lot and I did try to use undo when it happened. I was surprised it wasn't there, given my event log is being streamed to the UI.

**Comment by EricWittmann** [comment]
Yeah I'm restoring Undo/Redo as we speak.  It worked until I added multi-user collaboration.  When that was added, it make Undo/Redo a little trickier. :)

**Comment by sherl0cks** [comment]
Given the nature of this effort, I think undo/redo is a fine stop gap for this issue.
